bin/expo-setup.js

```diff
@@ -19,28 +19,24 @@ if (!isPlainObject(appJson) || !isPlainObject(appJson.expo)) {
   );
 }
 
-appJson.expo.experiments ||= {};
-// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access -- Set to empty object in previous line, if falsy
-appJson.expo.experiments.typedRoutes = true;
-
-appJson.expo.plugins = [
-  [
-    'expo-build-properties',
-    {
-      ios: {
-        newArchEnabled: true,
-      },
-      android: {
-        newArchEnabled: true,
-      },
-    },
-  ],
-];
```

Comment:
@ProchaLu is this equivalent with the new setting you mentioned comes by default:

> This PR updates the project configuration to align with changes introduced in Expo SDK 52. With SDK 52, the extra configuration steps required for SDK 51 are no longer needed. Running pnpm create expo-app@latest . now initializes the app.json file with:
> 
> - `expo.newArchEnabled = true`

Reply:
> A Boolean value that indicates whether the app should use the new architecture. Defaults to true.
https://docs.expo.dev/versions/latest/config/app/#newarchenabled

There is still the possibility to add the specific [iOS](https://docs.expo.dev/versions/latest/config/app/#newarchenabled-1) / [Android](https://docs.expo.dev/versions/latest/config/app/#newarchenabled-2) `newArchEnabled` settings, but we want to use the `newArchEnabled = true` in both.

Reply:
ok, that doesn't mention that it's the same as `expo-build-properties` anywhere, but it's probably the same, so let's stick with that

if you can easily verify that the app is running in the New Architecture, that would be good

Reply:
In the [docs](https://docs.expo.dev/guides/new-architecture/#enable-the-new-architecture-in-an-existing-project) from "Enable the New Architecture in an existing project": 
SDK 52: 
> To enable it on both Android and iOS, use the newArchEnabled at the root of the expo object in your app config. You can selectively enable it for a single platform by setting, for example, "android": { "newArchEnabled": true }.

SDK 51: 
> To enable it, you need to [install the expo-build-properties plugin](https://docs.expo.dev/versions/latest/sdk/build-properties#installation) and set newArchEnabled on target platforms.

Reply:
ok good, that's probably enough proof, thanks!